src/app/skills-page/skills-page.component.html

```diff
@@ -0,0 +1,63 @@
+<div class="main">
+    <p class="title">S K I L L S</p>
+
+    <div class="row container-fluid">
+
+        <div class="col container-fluid" *ngFor="let skill of skillsList| slice:0:4; let i = index">
+            <div class=" mb-3" style="max-width: auto;">
+                <div class="card-body text-white">
+                    <fa-icon class="logo" [icon]="[skill.type, skill.icon]"></fa-icon>
+                    <!-- <ng-template #t let-fill="fill">
+                        <i class="bi bi-star-fill">
+                            <i *ngIf="fill > 0" class="bi bi-star-fill filled" [style.width.%]="fill"></i>
+                        </i>
+                    </ng-template>
+                    <ngb-rating [(rate)]="skill.rate" [starTemplate]=" t" [readonly]="true" [max]="5"></ngb-rating> -->
```

Comment:
Zakomentowany kod się wyrzuca, jeśli byś potrzebowała do niego wrócić to masz go w gitcie